### ISSUE TITLE
Mod got worse after the update

### ISSUE THREAD

**Issue by axd1x8a** [oskardotglobal/gottagofast-fabric#9]
Even with all settings set to high values, traveling by boat on ice or flying on elytra is constantly interrupted, and the console shows logs from minecraft, not the mod, as if it doesn't work at all.
Minecraft version: 1.19.2
```json
{
	"hideConsoleWarnings": false,
	"playerLimit": 100000.0,
	"playerFallFlyingLimit": 100000.0,
	"vehicleLimit": 1500.0,
	"vehicleDistanceLimit": 100000.0
}
```

**Comment by oskardotglobal** [comment]
Are you running v1.1.0? Does the log show that the mod is loaded?

**Comment by axd1x8a** [comment]
Yes, mod loads and sometimes shows its logs if values set to default, but most of the time all movement and logs are handled by the server

**Comment by oskardotglobal** [comment]
Could you send the log or what exactly it logs? Ideally upload to mclo.gs

**Comment by axd1x8a** [comment]
I've noticed now that problem appears more often when MSPT spikes.

And about the logs:
on startup it says that mod is loaded:
`[GottaGoFast] GottaGoFast Fabric by oskardotglobal loaded successfully!`
when player is flying on elytra:
`[Server thread/WARN]: axd1x8a moved too quickly! 9.708459258641938,-0.7684762024129839,-3.631729591395498`
boat on ice and default values:
`[GottaGoFast] Boat (vehicle of axd1x8a) moved too quickly! Moved by 7.140766059100486,0.0,-7.28074180519161 blocks in x,y,z direction. If you wish to increase the limit to exclude this, increase the limit to 103.99974114466627`
boat on ice and high values:
`[Server thread/WARN]: Boat (vehicle of axd1x8a) moved too quickly! -3.78455785496476,0.0,-13.445834695408848`

There are no errors in the console, btw.

**Comment by oskardotglobal** [comment]
I'll look into it

**Comment by aaronfranke** [comment]
I'm still experiencing this problem. With higher-than-vanilla limits, the mod behaves as if it's not installed, it just has the vanilla "moved too quickly" messages.

```
[05:44:43] [Server thread/WARN]: Heliguy4599 moved too quickly! -13.064999628067028,1.3399999618530245,13.064999628066971
[05:44:45] [Server thread/WARN]: Heliguy4599 moved too quickly! -13.064999628067028,1.3399999618530245,13.064999628066971
[05:44:48] [Server thread/WARN]: Heliguy4599 moved too quickly! -13.064999628067028,1.3399999618530245,13.064999628066971
[05:44:49] [Server thread/WARN]: Heliguy4599 moved too quickly! -13.064999628067028,1.3399999618530245,13.064999628066971
[05:45:57] [Server thread/WARN]: Heliguy4599 moved too quickly! -5.834024833917624,1.3399999618530245,13.064999628066971
[05:47:16] [Server thread/WARN]: Heliguy4599 moved too quickly! -1.3304686831734784,-20.653909764802876,2.1235554757364525
```

With the default settings, I do see the mod's custom messages being printed, so the mod is installed.

I'm using Fabric with `gottagofast-1.1.0.jar` and `owo-lib-0.11.2+1.20.jar`.

**Comment by MightyBunch** [comment]
Can confirm, completely not working on v1.1.0 (1.20.1) if using not default values in config.
